src/styles/blocks/main-page.scss

```
@@ -0,0 +1,25 @@
+@use "../utilis/_variables" as *;
+@use "../utilis/_extend";
+
+.main-page {
+  display: flex;
+  justify-content: center;
+
+  &__container {
+    display: flex;
+    // justify-content: center;
+    flex-wrap: wrap;
+    max-width: #{$field-size};
```

I believe doing so is enough
```suggestion
    max-width: $field-size;
```